test_scripts/Polices/External_UCS/017_ATF_P_TC_PTU_without_disallowed_by_external_consent_entities_off_struct.lua

```diff
@@ -0,0 +1,120 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PTU without "disallowed_by_external_consent_entities_off struct
+--
+-- Description:
+-- In case:
+-- SDL receives PolicyTableUpdate without “disallowed_by_external_consent_entities_off:
+-- [entityType: <Integer>, entityId: <Integer>]” -> of "<functional grouping>"
+-- -> from "functional_groupings" section
+-- SDL must:
+-- a. consider this PTU as valid (with the pre-conditions of all other valid PTU content)
+-- b. do not create this "disallowed_by_external_consent_entities_off" field
+-- of the corresponding "<functional grouping>" in the Policies database.
+--
+-- Preconditions:
+-- 1. Start SDL (make sure 'disallowed_by_external_consent_entities_off' section is omitted in PreloadedPT)
```

Comment:
@istoimenova, Let me clarify. 
Req-t:
```
In case
SDL receives PolicyTableUpdate without “disallowed_by_external_consent_entities_off: [entityType: <Integer>, entityId: <Integer>]” -> of "<functional grouping>" -> from "functional_groupings" section
SDL must
a. consider this PTU as valid (with the pre-conditions of all other valid PTU content)
b. do not create this "disallowed_by_external_consent_entities_off" field of the corresponding "<functional grouping>" in the Policies database.
```

So the check is:
1. Perform PTU without _disallowed_by_external_consent_entities_off_ section
2. Check that final status is UP_TO_DATE
3. Check that section "disallowed_by_external_consent_entities_off" is omitted in Local Policy Table